.golangci.yml

```diff
@@ -42,3 +42,55 @@ linters-settings:
     suggest-new: true
   misspell:
     locale: US
+  depguard:
+    rules:
```

Comment:
Can't claim I understand the logic behind these exceptions, other than
"grab and override whatever the linter is currently complaining about"...

Reply:
Afaik `depguard` is just one of the linters that golangci-lint runs, and this provides explicit configuration for it. See https://github.com/OpenPeeDeeP/depguard